packages/muelu/src/Utils/MueLu_AggregateQualityEstimateFactory_def.hpp

```diff
@@ -64,13 +64,13 @@ RCP<const ParameterList> AggregateQualityEstimateFactory<Scalar, LocalOrdinal, G
 }
 
 template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-void AggregateQualityEstimateFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build(Level& currentLevel) const {
-  FactoryMonitor m(*this, "Build", currentLevel);
+void AggregateQualityEstimateFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build(Level& fineLevel, Level& coarseLevel) const {
+  FactoryMonitor m(*this, "Build", fineLevel);
 
-  RCP<Matrix> A              = Get<RCP<Matrix>>(currentLevel, "A");
-  RCP<Aggregates> aggregates = Get<RCP<Aggregates>>(currentLevel, "Aggregates");
+  RCP<Matrix> A              = Get<RCP<Matrix>>(fineLevel, "A");
+  RCP<Aggregates> aggregates = Get<RCP<Aggregates>>(fineLevel, "Aggregates");
 
-  RCP<const Map> map = Get<RCP<const Map>>(currentLevel, "CoarseMap");
+  RCP<const Map> map = Get<RCP<const Map>>(fineLevel, "CoarseMap");
```

Comment:
Is `fineLevel` correct?  I assume so, otherwise the `Get` would fail.  It just looks strange.

Reply:
Yes, this is correct ... sometimes it's weird on which level things are stored...